crates/driver/src/domain/competition/solution/mod.rs

```diff
@@ -48,6 +48,7 @@ pub struct Solution {
     solver: Solver,
     weth: eth::WethAddress,
     gas: Option<eth::Gas>,
+    surplus_capturing_jit_order_owners: HashSet<eth::Address>,
```

Comment:
These addresses are nothing specific to each solution. I makes more sense to pass a reference of this into `Solution::score()` and `Solution::is_empty()` instead.